src/omni_epd/displays/inky_display.py

```diff
@@ -49,6 +48,10 @@ def __init__(self, deviceName, config):
         elif(dType == 'what'):
             deviceObj = self.load_display_driver(self.pkg_name, 'what')
             self._device = deviceObj.InkyWHAT(dColor)
+        elif(dType == 'auto'):
+            deviceObj = self.load_display_driver(self.pkg_name, 'auto')
+            self._device = deviceObj.auto()
+            dColor = 'color' if self._device.colour == 'multi' else self._device.colour

```

Comment:
```suggestion
            if self._device.colour == 'multi':
                from inky.inky_uc8159 import DESATURATED_PALETTE
                self.mode = dColor = InkyImpressionDisplay.mode
                self.modes_available = InkyImpressionDisplay.modes_available
                self.max_colors = InkyImpressionDisplay.max_colors
                self.palette_filter = DESATURATED_PALETTE
                _display = InkyImpressionDisplay._display
                clear = InkyImpressionDisplay.clear
            else:
                dColor = self._device.colour
```
Got inky impression working.